src/sql/workbench/services/notebook/browser/notebookServiceImpl.ts

```diff
@@ -407,8 +407,7 @@ export class NotebookService extends Disposable implements INotebookService {
 		if (!notebookUri) {
 			return undefined;
 		}
-		// The NotebookEditor will not be found if there is query or fragments attached to the URI
-		let uriString = notebookUri.with({ query: '', fragment: '' }).toString();
+		let uriString = notebookUri.toString();
```

Comment:
Do you still need to make this change? I would expect that reverting this back to what you had originally  would still work.

Reply:
Chatted with @Charles-Gagnon offline. I will work on getting more robust logic in a separate PR.